frontend/package.json

```diff
@@ -4,19 +4,21 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
+    "bootstrap": "^4.3.1",
     "prop-types": "^15.7.2",
     "react": "^16.8.4",
+    "react-bootstrap": "^1.0.0-beta.8",
```

Comment:
We actually have reactstrap installed further down in the dependencies, which also provides both bootstrap 4 and a react interface. We probably shouldn't use both of them.

Reply:
should I delete the that specific line from this package.json file?

Reply:
Yeah, I'd remove that one and the bootstrap line further up, since that should be provided by reactstrap.

Reply:
Actually, bootstrap may still be necessary, I'm just lying.

Reply:
OK, I changed my mind again after doing some research. lol We shouldn't need to specifically add bootstrap here.